instrumentation/opentelemetry-instrumentation-pyramid/tests/test_programmatic.py

```diff
@@ -184,6 +198,20 @@ def test_internal_exception(self):
         self.assertEqual(span_list[0].name, "/hello/{helloid}")
         self.assertEqual(span_list[0].kind, trace.SpanKind.SERVER)
         self.assertEqual(span_list[0].attributes, expected_attrs)
+        self.assertEqual(
+            span_list[0].status.status_code, trace.StatusCode.ERROR
+        )
+        self.assertEqual(span_list[0].status.description, "error message")
+
+        expected_error_event_attrs = {
+            exception_attributes.EXCEPTION_TYPE: "NotImplementedError",
+            exception_attributes.EXCEPTION_MESSAGE: "error message",
+        }
+        self.assertEqual(span_list[0].events[0].name, "exception")
+        self.assertLess(
```

Comment:
That is to assert that `expected_error_event_attrs` is a subset of `dict(span_list[0].events[0].attributes)`? Could we make it more explicit?

Reply:
Maybe it's just me not used to use `<` to compare dictionary items :sweat_smile:

Reply:
Unfortunately, `assertDictContainsSubset` got deprecated. :cry: Maybe I can add a comment to give a brief explanation.

Reply:
Great, thanks